src/main/java/com/aws/greengrass/logging/impl/config/PersistenceConfig.java

```diff
@@ -357,6 +358,10 @@ protected RollingFileAppender<ILoggingEvent> getAppenderForFile(Logger loggerToC
                 .toString());
         logFilePolicy.setMaxFileSize(new FileSize(fileSizeKB * FileSize.KB_COEFFICIENT));
         logFilePolicy.setMaxHistory(maxHistory);
+        // TODO - consider letting customers configure how often GG should check logs for rollover / cleanup
+        // Check every 0.5 second for now
+        logFilePolicy.setCheckIncrement(Duration.buildBySeconds(0.5));
```

Comment:
.5 seems far too aggressive. Before your changes it was every 10 seconds I believe. Let's do 2 seconds maybe.

Reply:
updated to 2 seconds